api/v1alpha1/validation.go

```diff
@@ -416,13 +416,18 @@ func (r *RolloutDeviceSelection) Validate() []error {
 }
 
 func (d *DisruptionBudget) Validate() []error {
+	var errs []error
 	if d == nil {
 		return nil
 	}
 	if d.MinAvailable == nil && d.MaxUnavailable == nil {
-		return []error{errors.New("at least one of [MinAvailable, MaxUnavailable] must be defined in disruption budget")}
+		errs = append(errs, errors.New("at least one of [MinAvailable, MaxUnavailable] must be defined in disruption budget"))
 	}
-	return nil
+	groupBy := lo.FromPtr(d.GroupBy)
```

Comment:
Should we also validate that the items are valid label keys (in terms of regex)?

Reply:
https://github.com/flightctl/flightctl/blob/e1b67c342a623015be4a7f0f333d3d5a344e6b6a/api/v1alpha1/openapi.yaml#L4408